design/FY2017/NFP-Single-Zone VAV Fan Control for PackagedTerminalUnits.md

```diff
@@ -0,0 +1,510 @@
+        \note dedicated outdoor air through air terminal single duct mixer object.
+    A7, \field Supply Air Fan Object Type
+        \required-field
+        \type choice
+        \key Fan:SystemModel
+        \key Fan:OnOff
+        \key Fan:ConstantVolume
+        \note Fan:ConstantVolume only works when continuous fan operation is used the entire
+        \note simulation (all supply air fan operating mode schedule values are greater than 0).
+        \note If any fan operating mode schedule values are 0 a Fan:SystemModel or Fan:OnOff object must be used.
+    A8, \field Supply Air Fan Name
+        \required-field
+        \type object-list
+        \object-list FansCVandOnOff
+        \note Needs to match in the fan object.
+    A9, \field Heating Coil Object Type
+        \required-field
+        \type choice
+        \key Coil:Heating:Fuel
+        \key Coil:Heating:Electric
+        \key Coil:Heating:Water
+        \key Coil:Heating:Steam
+        \note Select the type of heating coil.
+    A10, \field Heating Coil Name
+        \required-field
+        \type object-list
+        \object-list HeatingCoilName
+        \note Needs to match in the heating coil object.
+    A11, \field Cooling Coil Object Type
+        \required-field
+        \type choice
+        \key Coil:Cooling:DX:SingleSpeed
+        \key Coil:Cooling:DX:VariableSpeed
+        \key CoilSystem:Cooling:DX:HeatExchangerAssisted
+        \note Select the type of Cooling Coil.
+        \note Only works with Coil:Cooling:DX:SingleSpeed or
+        \note CoilSystem:Cooling:DX:HeatExchangerAssisted or
+        \note Coil:Cooling:DX:VariableSpeed.
+    A12, \field Cooling Coil Name
+        \required-field
+        \type object-list
+        \object-list CoolingCoilsDXSingleSpeed
+        \object-list CoolingCoilsDXVariableSpeed
+        \note Needs to match a DX cooling coil object.
+    A13, \field Fan Placement
+        \type choice
+        \key BlowThrough
+        \key DrawThrough
+        \default DrawThrough
+        \note Select fan placement as either blow through or draw through.
+    A14, \field Supply Air Fan Operating Mode Schedule Name
+        \type object-list
+        \object-list ScheduleNames
+        \note Enter the name of a schedule that controls fan operation. Schedule Name values of 0 denote
+        \note cycling fan operation (fan cycles with cooling or heating coil). Schedule Name values greater
+        \note than 0 denote constant fan operation (fan runs continually regardless of coil operation).
+    A15, \field Availability Manager List Name
+        \note Enter the name of an AvailabilityManagerAssignmentList object.
+        \type object-list
+        \object-list SystemAvailabilityManagerLists
+    A16, \field Design Specification ZoneHVAC Sizing Object Name
+        \note Enter the name of a DesignSpecificationZoneHVACSizing object.
+        \type object-list
+        \object-list DesignSpecificationZoneHVACSizingName
+
+
+    New Inputs:
```

Comment:
Glad these are at the end, and all have meaningful defaults, so no other IDD changes needed.  Unless...do you anticipate exposing this option in the HVACTemplate objects that create these components?

Reply:
I'll let @mjwitte make the decision for HVACTemplate. Sounds like it would be worthwhile.

Reply:
@rraustad Always good to keep up with new options in HVACTemplate.